simplesat/transaction.py

```diff
@@ -73,6 +78,44 @@ def to_simple_string(self):
 
         return "\n".join(lines)
 
+    def install(self, package):
```

Comment:
This was there before this PR, but I don't think install/remove/update should be public APIs. Can we rename them to `_*` ?